src/actions/Spread.js

```diff
@@ -40,14 +40,14 @@ var Spread = function (items, property, min, max, inc)
     {
         for (i = 0; i < items.length; i++)
         {
-            items[i][property] += i * step;
+            items[i][property] += i * step + min;
```

Comment:
Should `min` be initialized with a default value (perhaps 0?) to prevent `NaN` result when `min` is omitted?

Reply:
I believe min isn't an optional parameter (?)

Reply:
My apologies! You are correct.